### PR TITLE
Replace ListCommand with ViewProfilesCommand

### PR DESCRIPTION
**Pull request by ichigh0st**
JUnit tests are modified as well.

ViewProfilesCommand retains ListCommand's behavior of ignoring everything after the command. For example, `profile -v asdf` is still considered a valid ViewProfilesCommand.

### CODE REVIEW

File: src/test/java/seedu/address/logic/LogicManagerTest.java
Comment (lesterong) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:69]: Minor nit
```suggestion
        String viewProfilesCommand = ViewProfilesCommand.COMMAND_WORD + " " + PREFIX_OPTION 
                + ViewProfilesCommand.COMMAND_OPTION;
```

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java
Comment (lesterong) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:90]: Perhaps it would be better to shift this test into `ProfileCommandParserTest` instead, since it's not exactly under the `AddressBookParser` anymore.

**Review by lesterong — COMMENTED** [review]
Works well, good job!

File: src/main/java/seedu/address/logic/parser/profile/ViewProfilesCommandParser.java
Comment (KSHan29) [inline src/main/java/seedu/address/logic/parser/profile/ViewProfilesCommandParser.java:21]: Perhaps this could be declared as a `static final` variable since the value will not change for each execution?

**Review by KSHan29 — APPROVED** [review]
Just a small nit. Tested it and worked well, good job!

Should be good to merge after fixing possible merge conflicts.

**Pull request merged**
